code/modules/surgery/organs_internal.dm

```diff
@@ -30,12 +29,24 @@
 		var/obj/item/organ/external/affected = H.get_organ(user.zone_sel.selecting)
 		if(affected && (affected.status & ORGAN_ROBOT))
 			return 0
-		if((target.get_species() == "Machine"))
+		if(target.get_species() == "Machine")
 			return 0
-		if((target.get_species() == "Diona"))
+		if(target.get_species() == "Diona")
 			return 0
```

Comment:
These surgeries will mess up if you graft organic limbs on the two black-listed species - the ORGAN_ROBOT check already watches for robotic body parts, and you can screen out diona parts by making sure `encased` is a string.